about.html

```diff
@@ -0,0 +1,137 @@
+          the world and has received brilliant reviews every year.
+        </p>
+      </div>
+      <div class="contact text-center py-3">
+        <span class="contact lead">Please contact us per Email for any further questions</span><br>
+        <span class="contact lead">about Concert Festival 2021.</span><br>
+        <span class="contact lead fw-semibold text-decoration-underline">info@cfestival.org</span>
+      </div>
+    </section>
+    <section id="cf-campaign">
+      <div class="text-center">
+        <h2>Concert Festival 2023 Logo</h2>
+        <span class="orange-bar mx-auto"></span>
+      </div>
+      <div class="text-center my-4">
+        <span class="text-center">The logo of Concert Festival 2023 was decided</span><br>
+        <span class="text-center">through the logo competition held from 1.Sept to 3.Oct</span>
+        <div class="cf-logo my-2">
+          <img src="./assets/mobile/cf-logo-white.jpg" alt="Concert Festival Logo" width="300" height="160">
+        </div>
+        <hr>
+      </div>
+
+      <div class="text-center">
+        <h2>See The Past Concert Festival</h2>
+        <span class="orange-bar mx-auto"></span>
+        <div class="text-center ">
+          <span>Take a look at the last two Concert Festival which took place</span><br/>
+          <span>in Moscow and Cairo</span>
+        </div>
+        <div class="past-events d-md-flex my-md-3 mx-auto">
+          <div class="moscow-event text-light">
+            <h2 class="display-3 fw-semibold">2022</h2>
+            <p class="lead fw-semibold">Concert Festival 2022 in Moscow</p>
+          </div>
+          <div class="text-light cairo-event">
+            <h2 class="display-3 fw-semibold">2021</h2>
+            <p class="lead fw-semibold">Concert Festival 2022 in Cairo</p>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section id="sponsors" class="container-fluid mx-auto d-md-none">
+      <div class="d-flex flex-wrap gap-3 partners justify-content-center">
+        <div class="container container-md">
+          <h2 class="text-center text-light">Partners</h2>
+          <span class="orange-bar mx-auto"></span>
+        </div>
+        <div class="partners-logo">
+          <img class="mx-auto" src="assets/mobile/partners/microsoft.png" alt="microsoft logo" height="100" width="200">
+          <img class="mx-auto" src="./assets/mobile/partners/google.png" alt="google logo" width="200" height="85">
+          <img class="mx-auto" src="./assets/mobile/partners/pixar.png" alt="pixar logo" width="300" height="100">
+          <img class="mx-auto" src="./assets/mobile/partners/microverse.png" alt="microverse logo" width="400" height="99">
+        </div>
+      </div>
+    </section>
+  </main>
+  <footer class="container-fluid">
+    <div class="container text-center lead fw-semibold">
+      <p>&commat; 2023 Concert Festival All Rights Reserved</p>
+    </div>
```

Comment:
- [ ] In the footer section we are supposed to see a logo and some text. Kindly consider fixing that.

Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/203996578-83ca61f7-6d8c-4934-81b5-746a670e7bf6.png)

The template design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/203996676-c65be2cf-4a70-49f3-b843-d0de33e39ee6.png)